ui/components/multichain/detected-token-banner/detected-token-banner.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { DetectedTokensBanner } from './detected-token-banner';
+
+export default {
+  title: 'Components/Multichain/DetectedTokensBanner',
+  component: DetectedTokensBanner,
+  argTypes: {
+    setShowDetectedTokens: { control: 'func' },
```

Comment:
Do actions work here?
```suggestion
    setShowDetectedTokens: { action: 'setShowDetectedTokens' },
```

Reply:
Yes